### PR TITLE
Add projects

### PR DESCRIPTION
**Pull request by AdityaHirapara**
https://adityahirapara.github.io/wdlsvnit.github.io/projects

### CODE REVIEW

File: projects/index.html [hunk 1]
```diff
@@ -47,14 +47,57 @@
...
-							<h2 class=mdl-card__title-text>Omeclone</h2>
+					<a href="https://paper.webdevlabs.in/" style="text-decoration: none;">
+						<div class="demo-card-wide mdl-card mdl-shadow--2dp">
+							<div class="mdl-card__supporting-text">
+								<h2 class=mdl-card__title-text>Paper-Extension</h2>
+							</div>
+							<div class=mdl-card__title style="background: url(../images/paperext.png) center; background-size: cover;">
+							</div>
+							<div class=mdl-card__supporting-text>
+								A chrome extension to sync the highlighted text from web to a dropbox paper.
```
Comment (ajayns) [inline projects/index.html]: Can this be "A chrome extension to save selected text from a webpage to dropbox paper"? I believe that makes it more clear.

File: projects/index.html [hunk 2]
```diff
@@ -47,14 +47,57 @@
...
+							<div class=mdl-card__supporting-text>
+								An opportunity for geeks to anonymously propose their crush. If your crush happens to like you as well, you both get connected over a phone call for 1 minute to express your feelings.
+							</div>
+						</div>
+					</a>
+					<br><br>
+					<a href="http://papergenie.herokuapp.com/" style="text-decoration: none;">
+						<div class="demo-card-wide mdl-card mdl-shadow--2dp">
+							<div class="mdl-card__supporting-text">
+								<h2 class=mdl-card__title-text>PaperGinie</h2>
```
Comment (ajayns) [inline projects/index.html]: Fix the typo here: `PaperGenie`

**Review by ajayns — COMMENTED** [review]
Other than the minor changes I mentioned everything looks good, do the needful and let's merge this :)

File: projects/index.html (hunk 1 above)
Comment (AdityaHirapara) [reply projects/index.html]: sounds good!

**Comment by lunaticmonk** [comment]
@ajayns @AdityaHirapara Are we good to merge this?

**Comment by AdityaHirapara** [comment]
Yes!

**Pull request merged**